addons/hr_holidays/models/hr_employee.py

```diff
@@ -150,13 +150,13 @@ def _compute_show_leaves(self):
                 employee.show_leaves = False
 
     def _search_absent_employee(self, operator, value):
-        holidays = self.env['hr.leave'].sudo().search([
+        holidays = self.env['hr.leave'].sudo()._search([
             ('employee_id', '!=', False),
             ('state', 'not in', ['cancel', 'refuse']),
             ('date_from', '<=', datetime.datetime.utcnow()),
             ('date_to', '>=', datetime.datetime.utcnow())
         ])
-        return [('id', 'in', holidays.mapped('employee_id').ids)]
+        return [('id', 'in', holidays.mapped('employee_id'))]
```

Comment:
You don't need `mapped`.  The existing API fits the operator `inselect`:
```suggestion
        return [('id', 'inselect', holidays.select('"hr_leave"."employee_id"'))]
```

Reply:
Note that we could improve `select` to automatically namespace a single identifier, so you could simply do `query.select('employee_id')`.